src/enums/zones.ts

```diff
@@ -43,93 +44,106 @@ export const COLOR_ZONE = {
 // For the summary text
 export const SUMMARY_TEXT: Zones = {
   [Level.LOW]: {
+    lowerLimit: 0,
     upperLimit: 0,
     name: 'All criteria met for reopening',
     color: COLOR_MAP.GREEN.BASE,
   },
   [Level.MEDIUM]: {
+    lowerLimit: 0,
     upperLimit: 0,
     name: 'Some criteria met for reopening',
     color: COLOR_MAP.ORANGE.BASE,
   },
   [Level.HIGH]: {
+    lowerLimit: 0,
     upperLimit: 0,
     name: 'No criteria met for reopening',
     color: COLOR_MAP.RED.BASE,
   },
   [Level.UNKNOWN]: {
+    lowerLimit: 0,
     upperLimit: 0,
     name: 'Unknown',
     color: COLOR_MAP.GRAY.BASE,
   },
 };
 
-// Case Growth Rate
 export const CASE_GROWTH_RATE: Zones = {
   [Level.LOW]: {
+    lowerLimit: 0.85,
     upperLimit: 1.05,
     name: 'Low',
     color: COLOR_MAP.GREEN.BASE,
   },
   [Level.MEDIUM]: {
+    lowerLimit: 1.05,
```

Comment:
Not a huge fan of having to duplicate these values between the different zones.  I feel like we probably want to move the graph bounds out of the zone definitions. Or have like a wrapper interface.

```
interface ChartZones {
  lowerLimit: number;
  upperLimit: number;
  zones: Zone[];
}
```

Reply:
yeah, I thought about that as well - I don't have much time today so feel free to update the PR, if this fix can wait I can polish a bit more this evening or tomorrow

Reply:
maybe we should have "chart" objects with title, subtitle, zones, upper and lower limits for the axes instead of just defining the zones and the chart data separately?